src/billable-services/cash-point/add-cash-point.modal.tsx

```diff
@@ -41,9 +44,9 @@ const AddCashPointModal: React.FC<AddCashPointModalProps> = ({ closeModal, onCas
   } = useForm<CashPointFormValues>({
     resolver: zodResolver(cashPointSchema),
     defaultValues: {
-      name: '',
-      uuid: '',
-      location: '',
+      name: cashPointToEdit?.name || '',
+      uuid: cashPointToEdit?.uuid || '',
+      location: cashPointToEdit?.location?.uuid || '',
```

Comment:
```suggestion
      name: cashPointToEdit?.name ?? '',
      uuid: cashPointToEdit?.uuid ?? '',
      location: cashPointToEdit?.location?.uuid ?? '',
```

Nullish coalescing is more appropriate here. If someone is editing a cash point and `name` is an empty string (perhaps cleared but not saved yet), || would still fallback to '', but it's semantically incorrect - you're not handling a "missing" value, you're handling an actual empty value.